software-containers/arcade-casa/README.md

```diff
@@ -0,0 +1,69 @@
+# Arcade CASA Container
+## Building A CASA container
+1. cd into the relevant folder with the build file or create a new folder if necessary
+2. update or create the relevant files, e.g. Makefile, Dockerfile, download.sh
```

Comment:
I'm not sure what point 2 means here.. are you talking about building what is in git or extending it for new versions?

Reply:
Under some circumstances, when creating a new version of CASA, a new folder would need to be created. In that case, we would need to create a Makefile and a Dockerfile.

Reply:
The documentation should only be about building what's currently in git.

Reply:
For some updates, we may need to update the download.sh file.

Reply:
For a new issue fix, we may need to create a file and place it in git.

Reply:
Modified wording.